docs/ui/customization/fragments/web/theming.md

```diff
@@ -1,3 +1,18 @@
+## Styling the amplify-authenticator
+
+You can directly style the `amplify-authenticator` component using CSS. Let's say that you wanted to update the layout by centering the component. You could use the following styling to achieve this.
+
+```css
+amplify-authenticator {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  min-height: 100vh;
```

Comment:
This line may not be the correct thing to do as it will set the entire component's height to 100vh. What we want is the containing app to be `100vh`.

Can we just add a note to make sure the app container in `100vh`?

Reply:
This works, but also ends up applying all styling to children so is not a good solution.